source/braille.py

```diff
@@ -2764,6 +2799,8 @@ def terminate():
 	global handler
 	handler.terminate()
 	handler = None
+	pre_speechCanceled.unRegister(clearBrailleRegions)

```

Comment:
```suggestion
	# noqa: F401 avoid module level import to prevent cyclical dependency
	# between speech and braille
	pre_speechCanceled.unRegister(clearBrailleRegions)
```